### PR TITLE
ENH: Amend TestQuantile::test_quantile_identification_equation (#9211)

### PR DESCRIPTION
**Pull request by chunweiyuan**
This PR amends the `TestQuantile::test_quantile_identification_equation` module in `numpy/lib/`, to better reflect the identification formula (Eq. 5 of https://doi.org/10.48550/arXiv.0912.0902) it refers to.

### CODE REVIEW

**Comment by seberg** [comment]
@lorentzenchr could you have a look?

File: numpy/lib/tests/test_function_base.py [hunk 1]
```diff
@@ -3683,30 +3683,32 @@ def test_quantile_scalar_nan(self):
     def test_quantile_identification_equation(self, method, alpha):
         # Test that the identification equation holds for the empirical
         # CDF:
-        #   E[V(x, Y)] = 0  <=>  x is quantile
+        #   dE[V(x, Y) * (x - y)]/dx = 0  <=>  x is quantile
```
Comment (lorentzenchr) [inline numpy/lib/tests/test_function_base.py:3686]: This is not correct. The old formula is correct, i.e. identification functions can be used as generalized L-Estimators.

**Review by lorentzenchr — COMMENTED** [review]
I think there is a slight misunderstanding. In my point of view the proposed changes worsens the current code.

File: numpy/lib/tests/test_function_base.py (hunk 1 above)
Comment (chunweiyuan) [reply numpy/lib/tests/test_function_base.py:3686]: I went by Eq. 6 of the Gneiting paper, and I presume you went by Eq. 13?

**Comment by chunweiyuan** [comment]
Upon further thought, I think the original test is better.  Please ignore/reject this PR.  Thanks.

File: numpy/lib/tests/test_function_base.py (hunk 1 above)
Comment (lorentzenchr) [reply numpy/lib/tests/test_function_base.py:3686]: Yes. 6 is for scoring functions, 13 for identification functions.

**Pull request closed without merging**